### ISSUE TITLE
Batch based training

### ISSUE THREAD

**Issue by mmderakhshani** [allanzelener/YAD2K#49]
Hi @allanzelener.
Thank you again for your amazing implementation. I have got one more question. Could you please tell me, does your loss function do batch processing? If yes, Could you please tell me, how do you prepare your `ground truth labels` data for batch processing? Let me tell you what I mean. Actually what I want to do is to create a tensor named `true_boxes` with shape `[batch, num_true_boxes, 5]` for training. As you can guess, Number of bbox in every picture is different. One image may have 4 bboxes, another may have 2 or another 3. So It seems that the `num_true_boxes` in every image has different value. So how did you solve this issue?
 
Thanks!

**Comment by allanzelener** [comment]
Just zero pad the bounding boxes for the images in the batch that have less than the maximum number of possible bounding boxes in the batch.
